Documentation/Migration/Index.rst

```diff
@@ -0,0 +1,29 @@
+..  include:: ../Includes.rst.txt
+
+..  _migration:
+
+=========
+Migration
+=========
+
+If you want to migrate from using Sphinx to render TYPO3 documentation to
+using the render-guides, the file :file:`Documentation/Settings.cfg` has to be
+replaced by an XML file, :file:`Documentation/guides.xml`.
+
+To facilitate migration the extension `t3docs/typo3-guides-cli` in this
+mono-repository comes with a Symfony Console Command to automatically migrate
+the outdated :file:`Settings.cfg`.
+
+With make
+=========
+
+..  code-block:: sh
+
+    make migrate-settings path=path/to/Documentation
+
+With ddev
+=========
+
+..  code-block:: sh
+
+    ddev exec packages/typo3-guides-cli/bin/typo3-guides migrate path/to/Documentation
```

Comment:
I'd prefer to advertise using either the makefile syntax or composer wrapper only, not introducint native calls to? So with either

ddev exec "make ENV=local migrate-settings ..."

Or

ddev composer make migrate-settings

(The other docs use the latter which is IMO clearer to read because it doesnt need the ENV=local part)

What do you think?

Reply:
You are right, changed it